### DISCUSSION TITLE
descriptionForMessaging() is missing in otel/internal/utils/spanattributes.go?

### DISCUSSION THREAD

**Discussion by yolkhovyy** [getsentry/sentry-go#639] [General]
Hello,  looks like a `descriptionForMessaging()` is missing in [otel/internal/utils/spanattributes.go](https://github.com/getsentry/sentry-go/blob/master/otel/internal/utils/spanattributes.go) - are there any plans to implement it?

Would be nice if it makes this two displayed in UI at the end:

- semconv.MessageTypeSent
- semconv.MessageTypeReceived

**Comment by tonyo** [comment]
Yes, we're tracking it in https://github.com/getsentry/sentry-go/issues/613, let's move the discussion there.

**Reply by yolkhovyy** [reply]
Thanks
